### PR TITLE
Stamp Duty bottom link update

### PR DESCRIPTION
**Pull request by akenzua**
[TP12656 ](https://maps.tpondemand.com/restui/board.aspx#page=board/5175634625349962976&appConfig=eyJhY2lkIjoiOUM2MjFGQzAzOEIzQTVDNDU1NjRCMUIxQ0FENkZBOTYifQ==&boardPopup=userstory/12656/silent)

This PR remove the duplication of "and" in the bottom link on stamp duty calculator.

### CODE REVIEW

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">It looks like this is a Rails engine, but no changes to <a href="https://github.com/moneyadviceservice/mortgage_calculator/blob/836f6f0b22b732995f2fa2335a44944a15bd6370/lib/mortgage_calculator/version.rb">lib/mortgage_calculator/version.rb</a> detected. Did you forget to bump the version?</td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: config/locales/stamp_duty.en.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ en:
         link: "https://www.moneyhelper.org.uk/en/homes/buying-a-home/land-transaction-tax-calculator-wales"
       scotland:
         title: Calculate Land and Buildings Transaction Tax for Scotland
-        link: "https://www.moneyhelper.org.uk/en/homes/buying-a-home/land-and-and-buildings-transaction-tax-calculator-scotland"
+        link: "https://www.moneyhelper.org.uk/en/homes/buying-a-home/land-and-buildings-transaction-tax-calculator-scotland"
```
Comment (davidtrussler) [inline config/locales/stamp_duty.en.yml:39]: This looks good. 

Although not explicitly stated in the TP card I think there will need to be a similar change made it he corresponding `.cy` file. Might be worth checking but it is probably also wrong.

Comment (akenzua) [reply config/locales/stamp_duty.en.yml:39]: Thanks

**Review by davidtrussler — APPROVED** [review]
OK, looks good! 👍

**Pull request merged**